vscode-extension/grammars/graphql.js.json

```diff
@@ -0,0 +1,67 @@
+{
+  "fileTypes": [
+    "js",
+    "jsx",
+    "mjs",
+    "cjs",
+    "es6",
+    "es",
+    "esm",
+    "ts",
+    "tsx",
+    "vue",
+    "svelte"
```

Comment:
Should we drop these?

Reply:
oh, didn't realize they were declared in both places.